substra/cli/interface.py

```diff
@@ -189,6 +198,19 @@ def click_option_expand(f):
     )(f)
 
 
+def click_global_conf_retry_timeout(f):
+    """Add timeout option to command."""
+    return click.option(
+        '--timeout', 'timeout',
+        type=click.INT,
+        expose_value=False,
+        default=DEFAULT_RETRY_TIMEOUT,
+        show_default=True,
+        callback=update_global_conf,
+        help='Set timeout (seconds) for retrying a call'
```

Comment:
This help text confused me a bit (I was not sure if this was actually a "delay" between each call). 

I think it would be more clear with:

```suggestion
        help='Max number of seconds the operation will be retried for'
```

What do you think?

Reply:
Yes it is clearer !